### PR TITLE
Test linethickness with unitless zero

### PR DESCRIPTION
**Pull request by rwlbuis**

### CODE REVIEW

File: mathml/presentation-markup/fractions/frac-linethickness-004-ref.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        /* FractionRuleThickness = 10000 * 1 / 1000 = 10px; */
+        font-family: "TestFont";
+        font-size: 1px;
+      }
+    </style>
+  </head>
+  <body>
+    <p>This test passes if you see no fraction bar.</p>
+    <math>
+      <mspace width="0px"/>
```
Comment (fred-wang) [inline mathml/presentation-markup/fractions/frac-linethickness-004-ref.html]: I don't think this mspace is needed for this test

File: mathml/presentation-markup/fractions/frac-linethickness-004-ref.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <mfrac>
```
Comment (fred-wang) [inline mathml/presentation-markup/fractions/frac-linethickness-004-ref.html]: it should be linethickness="0px" right?

**Review by fred-wang — CHANGES_REQUESTED** [review]

File: mathml/presentation-markup/fractions/frac-linethickness-004-ref.html (hunk 1 above)
Comment (rwlbuis) [reply mathml/presentation-markup/fractions/frac-linethickness-004-ref.html]: Yeah, not sure what I was thinking here, fixed.

File: mathml/presentation-markup/fractions/frac-linethickness-004-ref.html (hunk 2 above)
Comment (rwlbuis) [reply mathml/presentation-markup/fractions/frac-linethickness-004-ref.html]: Right, fixed.

**Review by fred-wang — APPROVED** [review]
LGTM

**Pull request merged**